paimon-web-api/src/main/java/org/apache/paimon/web/api/catalog/PaimonServiceFactory.java

```diff
@@ -26,15 +26,30 @@
 
 import org.apache.commons.lang3.StringUtils;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /** Paimon service factory. */
 public class PaimonServiceFactory {
 
-    public static PaimonService createFileSystemCatalogService(String name, String warehouse) {
+    public static PaimonService createFileSystemCatalogService(
+            String name, String warehouse, Map<String, String> catalogOptions) {
+        if (catalogOptions == null) {
+            catalogOptions = new HashMap<>();
```

Comment:
When `catalogOptions` is null, does the logic need to consider the filesystem type?

Reply:
If catalogOptions is empty, it will be treated as the local file system